src/VS.Web.CG.Mvc/Templates/ControllerGenerator/ApiControllerWithContext.cshtml

```diff
@@ -56,8 +56,8 @@ namespace @Model.ControllerNamespace
         }
 
         // PUT: @routePrefix/5
-        // To protect from overposting attacks, enable the specific properties you want to bind to, for
-        // more details, see https://go.microsoft.com/fwlink/?linkid=2123754.
+        // To protect from overposting attacks, enable the specific properties you want to bind to.
```

Comment:
// To protect from overposting attacks, see https://go.microsoft.com/fwlink/?linkid=2123754

properties you want to bind to is not used in API binding.
Ending the sentence with a period is correct grammar, but it makes it harder to copy/paste.

Generally we don't take PR's on slight improvements to comments, but fix this and it's worth taking.

Reply:
Thanks for the feedback an corrections @Rick-Anderson ! I've updated the patch according to your recommendations.